### ISSUE TITLE
Crash when parsing nested at rules (@import in media query)

### ISSUE THREAD

**Issue by timbru31** [shellscape/postcss-less#103]
* Node Version: v8.11.1
* NPM Version: v5.6.0
* postcss-less Version: v1.1.5

### LESS

```less
@media screen {
	@import "basic";
}
```

### JavaScript

n/a

### Errors 

```
TypeError: this[node.type] is not a function
    at Stringifier.stringify (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringifier.js:33:24)
    at Stringifier.body (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringifier.js:93:18)
    at Stringifier.block (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringifier.js:103:18)
    at Stringifier.atrule (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringifier.js:74:18)
    at Stringifier.stringify (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringifier.js:33:24)
    at stringify (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/stringify.js:14:9)
    at AtRule.toString (/Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/node.js:197:9)
    at check (/Users/timbru/work/ume/door-to-door-webapp/node_modules/stylelint/lib/rules/block-closing-brace-empty-line-before/index.js:58:41)
    at /Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/container.js:296:28
    at /Users/timbru/work/ume/door-to-door-webapp/node_modules/postcss-less/node_modules/postcss/lib/container.js:148:26
```

### Expected Behavior

Parsing should work.

### Actual Behavior

Parsing crashes.

### How can we reproduce the behavior?

`npx stylelint --syntax less '{src,styleguide}/**/*.less'` with the example less above.
PS: I am not sure if this is an upstream (postcss) issue or related to the postcss-less parsing.

**Comment by shellscape** [comment]
Hard to say who's the culprit here. First stop should be `stylelint` if you're not interested in submitting a PR to fix the issue. those folks are usually quick to patch something like this.

**Comment by shellscape** [comment]
Good news: this is resolved by the work done in #113
